resources/js/Pages/Guest/VerifyEmail.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import { Head, useForm } from '@inertiajs/vue3'
+import { inject, onMounted, type Ref } from 'vue'
+import type Header from '@/components/Common/Header.vue'
+
+const form = useForm({})
```

Comment:
Did you need a useForm here if you don't set any values inside. You can use just a router from inertia. 
https://inertiajs.com/manual-visits

Reply:
I'm using `form.processing` in this file, it is only available in `useForm`

Reply:
You can also use some event callbacks to handle 'processing' -> https://inertiajs.com/manual-visits.